sorc/checkout.sh

```diff
@@ -32,9 +32,7 @@ fi
 echo gsi checkout ...
 if [[ ! -d gsi.fd ]] ; then
     rm -f ${topdir}/checkout-gsi.log
-#    git clone --recursive --branch gfsda.v16.3.12 https://github.com/NOAA-EMC/GSI.git gsi.fd >> ${topdir}/checkout-gsi.log 2>&1
-# Check out develop for now
-    git clone --recursive https://github.com/NOAA-EMC/GSI.git gsi.fd >> ${topdir}/checkout-gsi.log 2>&1
+    git clone --recursive --branch gfsda.v16.3.25 https://github.com/NOAA-EMC/GSI.git gsi.fd >> ${topdir}/checkout-gsi.log 2>&1
```

Comment:
Please update the model tag on line 26 to be `GFS.v16.3.25`. Please also update `Externals.cfg` to reflect the new component tags. Thanks!

Reply:
Done

Reply:
I don't see the model tag updated in here nor updates to `Externals.cfg`. Please do those and I think we'll be good to merge this. Thanks!